testsuite/generate-format/tests/staged/api.yaml

```diff
@@ -194,11 +194,6 @@ Identifier:
 Module:
   STRUCT:
     - code: BYTES
-ModuleBundle:
```

Comment:
@wrwg @runtian-zhou do we want to keep `ModuleBundle` APIs and encapsulation, or switch to Vec<Vec<u8>> or similar?

Reply:
I don't think you can remove this for it to be compatible until the current code stops using it for the code module bundle